ebpf-profiling.graphqls

```diff
@@ -225,7 +272,7 @@ extend type Query {
     # query eBPF profiling data for prepare create task
     queryPrepareCreateEBPFProfilingTaskData(serviceId: ID!): EBPFProfilingTaskPrepare!
     # query eBPF profiling task list
-    queryEBPFProfilingTasks(serviceId: ID, serviceInstanceId: ID, targets: [EBPFProfilingTargetType!]): [EBPFProfilingTask!]!
+    queryEBPFProfilingTasks(serviceId: ID, serviceInstanceId: ID, targets: [EBPFProfilingTargetType!], triggerType: EBPFProfilingTriggerType, duration: Duration): [EBPFProfilingTask!]!
```

Comment:
``` suggestion
    # query `triggerType == FIXED_TIME` when triggerType is absent
    queryEBPFProfilingTasks(serviceId: ID, serviceInstanceId: ID, targets: [EBPFProfilingTargetType!], triggerType: EBPFProfilingTriggerType, duration: Duration): [EBPFProfilingTask!]!
```

Question about `duration`. what is it about?

Reply:
`duration` provide the filter for which task start time is matched.

Reply:
Are you going to query tasks on Duration this time? This looks like a little conflict with manual profiling. How about we list all, and ask UI to consider paging directly?

Reply:
added the paging parameter, and default list all if paging not provide.